tox.ini

```diff
@@ -4,11 +4,13 @@ envlist = py{27,36}, flake8, docs
 [testenv]
 recreate = True
 passenv = DISPLAY MOZ_HEADLESS TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+
+setenv = PIPENV_IGNORE_VIRTUALENS=1
 deps = -rpipenv.txt
 commands =
     pipenv install --dev --skip-lock
     pipenv run pytest --driver Firefox --cov --html results/{envname}.html {posargs}
-    coveralls
+    pipenv run coveralls
```

Comment:
I just noticed that this fails locally. If you prefix it with `- ` then tox will ignore the failure. See [the docs](https://tox.readthedocs.io/en/latest/config.html#confval-commands=ARGVLIST) for more details.

Reply:
Fixed; thanks (and sorry!)